core/build.gradle

```diff
@@ -97,6 +97,7 @@ dependencies {
     androidTestImplementation "androidx.test.espresso:espresso-core:$espressoVersion"
     androidTestImplementation 'androidx.test:runner:1.2.0'
     androidTestImplementation 'androidx.test:rules:1.2.0'
+    implementation 'org.jetbrains:annotations:15.0'
```

Comment:
There is a NonNull annotation in the Androidx library. We don't need the jetbrains version.